bands/templates/bands/agreement-base.html

```diff
@@ -84,7 +86,9 @@
     </h3>
     <p>
         {% block band_text_parking_inner %}
-        To be included with hotel room for <b>{{ band.vehicles }} vehicle{{ band.vehicles|pluralize }}</b>.
+        To be included
```

Comment:
Business logic sanity check: this text could end up saying "To be included for 4 band vehicles." Does this mean we'll provide parking with or without a hotel room?